Metalama.Framework/Advising/AdviserExtensions.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) SharpCrafters s.r.o. See the LICENSE.md file in the root directory of this repository root for details.
+
+using Metalama.Framework.Aspects;
+using Metalama.Framework.Code;
+using Metalama.Framework.Code.DeclarationBuilders;
+using System;
+
+namespace Metalama.Framework.Advising;
+
+public static class AdviserExtensions
```

Comment:
I assume support for other member types will be added later?

Reply:
Yeah, there's a task. It's copy-paste from IAdviceFactory.